issue_metrics.py

```diff
@@ -236,19 +234,19 @@ def get_per_issue_metrics(
                 )
             if labels and env_vars.hide_label_metrics is False:
                 issue_with_metrics.label_metrics = get_label_metrics(issue, labels)
-            if env_vars.hide_time_to_close is False:
-                if issue.state == "closed":  # type: ignore
-                    if pull_request:
-                        issue_with_metrics.time_to_close = measure_time_to_merge(
-                            pull_request, ready_for_review_at
-                        )
-                    else:
-                        issue_with_metrics.time_to_close = measure_time_to_close(
-                            issue, None
-                        )
-                    num_issues_closed += 1
-                elif issue.state == "open":  # type: ignore
-                    num_issues_open += 1
+            if env_vars.hide_time_to_close is False and issue.state == "closed":  # type: ignore
+                if pull_request:
+                    issue_with_metrics.time_to_close = measure_time_to_merge(
+                        pull_request, ready_for_review_at
+                    )
+                else:
+                    issue_with_metrics.time_to_close = measure_time_to_close(
+                        issue, None
+                    )
+            if issue.state == "closed":  # type: ignore
+                num_issues_closed += 1
+            elif issue.state == "open":  # type: ignore
+                num_issues_open += 1
```

Comment:
Logically equivalent but is this more readable/performant?
```suggestion
            if issue.state == "closed":  # type: ignore
                num_issues_closed += 1
                if env_vars.hide_time_to_close is False:
                    if pull_request:
                        issue_with_metrics.time_to_close = measure_time_to_merge(
                            pull_request, ready_for_review_at
                        )
                    else:
                        issue_with_metrics.time_to_close = measure_time_to_close(
                            issue, None
                        )
            elif issue.state == "open":  # type: ignore
                num_issues_open += 1
```

Reply:
same thing, I do think it is more readable but think `if not env_vars.hide_time_to_close` is more idiomatic.  I might be wrong. 🤔 wdyt?